source/lovelace/views.markdown

```diff
@@ -11,17 +11,17 @@ footer: true
 
 These are exactly as before, tab views with icons or text that help you manage large dashboards with many entities. The views have now deep links like `/lovelace/0`. You can also assign your own [custom ids](/lovelace/views/#custom-id).
 
-- Using custom id in view, for [nicer navigation paths](/lovelace/views/#custom-id) in URLs
-- Using [icons](/lovelace/views/#icons) instead of text
-- Using a card to [fill a complete view](/lovelace/views/#panel-view), just like panels
-- Using [themes](/lovelace/views/#themes) in views
+- Using custom id in view, for [nicer navigation paths](/lovelace/views/#custom-id) in URLs.
+- Using [icons](/lovelace/views/#icons) instead of text.
+- Using a card to [fill a complete view](/lovelace/views/#panel-view), just like panels.
+- Using [themes](/lovelace/views/#themes) in views.
 
 | Name | Type | Default | Description
```

Comment:
Should we change to use configuration tags as the other lovelace documentation?

```
{% configuration %}
title:
  required: false
  description: Text title of the view
  type: string
id:
  required: false
  description: The id to use in URL path of this view
  type: string
icon:
  required: false
  description: The material design icon for the view, uses this instead of title
  type: string
panel:
  required: false
  description: Marks view as a panel reusing the first card in the list
  type: boolean
{% endconfiguration %}
```

Reply:
You probably could, but that is out of the scope of this PR. 
I mainly focussed on spelling & grammar.